openverse-api/test/v1_integration_test.py

```diff
@@ -221,7 +221,7 @@ def test_creator_quotation_grouping():
     # Did we find only Claude Monet works, or did his lesser known brother Jim
     # Monet sneak into the results?
```

Comment:
The comment is outdated.
```suggestion
    # Did we find only William Ford Stanley works, or also by others?
```